client/layout/masterbar/logged-in.jsx

```diff
@@ -491,24 +492,20 @@ class MasterbarLoggedIn extends Component {
 	}
 
 	render() {
-		const { isCheckout, isCheckoutPending } = this.props;
+		const { isInEditor, isCheckout, isCheckoutPending } = this.props;
 		const { isMobile } = this.state;
 
+		const currentSegment = 10; //percentage of users that will see the Help Center, not the FAB
+		const userSegment = this.props.user.ID % 100;
+		const userAllowedToHelpCenter = userSegment < currentSegment && this.props.locale === 'en';
```

Comment:
This logic seems very help-center related and probably shouldn't live in the primary masterbar `render()` method.

Reply:
Luckily this part is temporary, and will be removed in 2 weeks when we release the Help Center to 100% of the users.
But of course, we could move this logic to a function inside the HelpCenter package, if this is what you mean 😄